source/kraken/tests/realtime_test.cpp

```diff
@@ -2740,3 +2740,380 @@ BOOST_AUTO_TEST_CASE(should_get_correct_base_stop_time_with_lollipop_II) {
     BOOST_CHECK_EQUAL(stoptime_rltm_B_bis.get_base_stop_time(), &vj->stop_time_list[3]);
     BOOST_CHECK_EQUAL(stoptime_rltm_D.get_base_stop_time(), &vj->stop_time_list[4]);
 }
+
+BOOST_AUTO_TEST_CASE(add_new_trip) {
+
+    using year = navitia::type::ValidityPattern::year_bitset;
+
+    ed::builder b("20190101");
+    auto& pt_data = *b.data->pt_data;
+
+    b.sa("A", 0, 0, true, true);
+    b.sa("B", 0, 0, true, true);
+    b.sa("C", 0, 0, true, true);
+    b.sa("D", 0, 0, true, true);
+    b.sa("E", 0, 0, true, true);
+    b.sa("F", 0, 0, true, true);
+    b.sa("G", 0, 0, true, true);
+    b.sa("H", 0, 0, true, true);
+    b.sa("I", 0, 0, true, true);
+    b.sa("J", 0, 0, true, true);
+
+    b.vj("1").uri("vj:1")
+                 ("stop_point:A", "08:00"_t)
+                 ("stop_point:B", "08:30"_t)
+                 ("stop_point:C", "09:00"_t)
+                 ("stop_point:D", "09:30"_t);
+
+    // Add company
+    std::string comp_name = "comp_name";
```

Comment:
The fact there is no method to create only simple Company or Physical_mode like Stop_time...
I have just created my way because I want to control all process.
But I think, this is a good remark to create a JIRA ticket within the technical debt to improve the build_helper

Reply:
To be managed through JIRA ticket test refactor (+ build_helper improvement)

Reply:
JIRA : https://jira.kisio.org/browse/NAVP-1228